### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
List of changes:
- Add changelog link to hex module info page
- Fix logo not showing in HTML page
- Add logo to table of content
- Fix typos
- Fix zstd links in html doc
- Badges and more badges!

### CODE REVIEW

**Comment by fhunleth** [comment]
I like it. Thanks!

File: README.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
 # Bakeware
 
-[![Hex version](https://img.shields.io/hexpm/v/bakeware.svg "Hex version")](https://hex.pm/packages/bakeware)
 [![CircleCI](https://circleci.com/gh/bake-bake-bake/bakeware.svg?style=svg)](https://circleci.com/gh/bake-bake-bake/bakeware)
+[![Hex version](https://img.shields.io/hexpm/v/bakeware.svg "Hex version")](https://hex.pm/packages/bakeware)
+[![Hex Docs](https://img.shields.io/badge/hex-docs-lightgreen.svg)](https://hexdocs.pm/bakeware/)
+[![Total Download](https://img.shields.io/hexpm/dt/bakeware.svg)](https://hex.pm/packages/bakeware)
+[![License](https://img.shields.io/hexpm/l/bakeware.svg)](https://hex.pm/packages/bakeware)
+[![Last updated](https://img.shields.io/github/last-commit/bake-bake-bake/bakeware.svg)](https://github.com/bake-bake-bake/bakeware/commits/master)
```
Comment (jjcarstens) [inline README.md:8]:
```suggestion
[![Last updated](https://img.shields.io/github/last-commit/bake-bake-bake/bakeware.svg)](https://github.com/bake-bake-bake/bakeware/commits/main)
```

**Review by jjcarstens — APPROVED** [review]

**Comment by fhunleth** [comment]
@kianmeng We didn't know you were online and went ahead and fixed the `master` -> `main` reference.

**Comment by kianmeng** [comment]
> @kianmeng We didn't know you were online and went ahead and fixed the `master` -> `main` reference.

@fhunleth Go ahead. That the purpose of `Allow edits by maintainers` for opened PR. :smile:

**Pull request merged**